### PR TITLE
chore: update tech spec

### PR DESCRIPTION
**Pull request by Jcahilltorre**
### Proposed changes

- Updates the tables in the tech spec guide to make full use of the bootstrap-table shortcode in Hugo
- Updates some sentences to be more concise and simplify the language.
- Updates references to the product ( to "NGINX Ingress Controller")

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/technical-specifications.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ docs: "DOCS-617"
...
+Release 1.12 supports the Ingress v1beta1 API and continues to receive security fixes to support environments running Kubernetes versions older than 1.19. The v1beta1 Ingress API was deprecated in Kubernetes release 1.19, and removed in the Kubernetes 1.22.
 
 ## Supported Kubernetes Versions
 
-We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes platforms at each release, and the [release notes](/nginx-ingress-controller/releases) list which platforms were tested. We will provide technical support for the NGINX Ingress Controller (NIC) on any Kubernetes platform that is currently supported by its provider and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
+We explicitly test NGINX Ingress Controller on a range of Kubernetes platforms for each release, and we list them in the [release notes](/nginx-ingress-controller/releases). We provide technical support for NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider, and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
```
Comment (travisamartin) [inline docs/content/technical-specifications.md:21]: Here and throughout, update links to "relref" where possible.

File: docs/content/technical-specifications.md [hunk 2]
```diff
@@ -38,64 +38,69 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
...
+{{% /bootstrap-table %}}
 
 ### Images with NGINX Plus
 
-NGINX Plus images include NGINX Plus R30.
+_NGINX Plus images include NGINX Plus R30._
+
+#### **F5 Container registry**
 
 NGINX Plus images are available through the F5 Container registry `private-registry.nginx.com` - see [Using the NGINX IC Plus JWT token in a Docker Config Secret](/nginx-ingress-controller/installation/using-the-jwt-token-docker-secret) and [Pulling the NGINX Ingress Controller image]({{< relref "installation/nic-images/pulling-ingress-controller-image" >}}).
```
Comment (travisamartin) [inline docs/content/technical-specifications.md:65]: Update link to relref

File: docs/content/technical-specifications.md [hunk 3]
```diff
@@ -38,64 +38,69 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
...
 |Ubi-based image with NGINX App Protect WAF | ``redhat/ubi8`` | NGINX App Protect WAF and NGINX Plus JavaScript module | `nginx-ic-nap/nginx-plus-ingress:3.2.1-ubi` | amd64 |
 |Ubi-based image with NGINX App Protect DoS | ``redhat/ubi8`` | NGINX App Protect DoS and NGINX Plus JavaScript module | `nginx-ic-dos/nginx-plus-ingress:3.2.1-ubi` | amd64 |
-|Ubi-based image with NGINX App Protect WAF and DoS | ``redhat/ubi8`` | NGINX App Protect WAF and DoS, NGINX Plus JavaScript module | `nginx-ic-nap-dos/nginx-plus-ingress:3.2.1-ubi` | amd64 |
-{{% /table %}}
+|Ubi-based image with NGINX App Protect WAF and DoS | ``redhat/ubi8`` | NGINX App Protect WAF and DoS<br><br>NGINX Plus JavaScript module | `nginx-ic-nap-dos/nginx-plus-ingress:3.2.1-ubi` | amd64 |
+{{% /bootstrap-table %}}
+
+#### **AWS Marketplace**
 
 We also provide NGINX Plus images through the AWS Marketplace. Please see [Using the AWS Marketplace Ingress Controller Image](/nginx-ingress-controller/installation/using-aws-marketplace-image/) for details on how to set up the required IAM resources in your EKS cluster.
```
Comment (travisamartin) [inline docs/content/technical-specifications.md:84]: Update link to relref

File: docs/content/technical-specifications.md [hunk 4]
```diff
@@ -38,64 +38,69 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
...
+|Debian-based image with NGINX App Protect DoS | ``debian:11-slim`` | NGINX App Protect DoS<br><br>NGINX Plus JavaScript and OpenTracing modules<br><br>OpenTracing tracers for Jaeger<br><br>Zipkin and Datadog | [F5 NGINX Ingress Controller with F5 NGINX App Protect WAF and DoS](https://aws.amazon.com/marketplace/pp/prodview-yltaqwzwrnhco) | amd64 |
+|Debian-based image with NGINX App Protect WAF and DoS | ``debian:11-slim`` | NGINX App Protect WAF and DoS<br><br>NGINX Plus JavaScript and OpenTracing modules<br><br>OpenTracing tracers for Jaeger<br><br>Zipkin and Datadog | [F5 NGINX Ingress Controller with F5 NGINX App Protect DoS](https://aws.amazon.com/marketplace/pp/prodview-sghjw2csktega) | amd64 |
+{{% /bootstrap-table %}}
 
+#### **Google Cloud Marketplace**
 We also provide NGINX Plus images through the Google Cloud Marketplace. Please see [Using GCP Marketplace Ingress Controller](/nginx-ingress-controller/installation/using-gcp-marketplace-package/) for details on how use them.
```
Comment (travisamartin) [inline docs/content/technical-specifications.md:96]: Update link to relref

**Review by travisamartin — APPROVED** [review]

**Comment by travisamartin** [comment]
Looks good. Just a few comments to change some links to relref.

**Review by vepatel — APPROVED** [review]

File: docs/content/technical-specifications.md [hunk 5]
```diff
@@ -106,8 +111,4 @@ You can customize an existing Dockerfile or use it as a reference to create a ne
 
 ## Supported Helm Versions
 
-The Ingress Controller supports installation via Helm 3.0+.
-
-## Recommended Hardware
-
-See the [Sizing guide](https://www.nginx.com/resources/datasheets/nginx-ingress-controller-kubernetes-sizing-guide/) for recommendations.
+NGINX Ingress Controller supports [installation via Helm 3.0+]({{< relref "/installation/installing-nic/installation-with-helm.md" >}}).
```
Comment (lucacome) [inline docs/content/technical-specifications.md:114]:
```suggestion
NGINX Ingress Controller supports [installation] via Helm 3.0+({{< relref "/installation/installing-nic/installation-with-helm.md" >}}).
```

Comment (Jcahilltorre) [reply docs/content/technical-specifications.md:114]: I think that change won't work.
image

I am opening a separate PR to change that link , as this is now merged to the revamp branch

Comment (lucacome) [reply docs/content/technical-specifications.md:114]: ah sorry I meant to move the entire link to just `installation`

**Pull request merged**